server/database/queries/messages/addChannel.js

```diff
@@ -1,9 +1,23 @@
 const dbConnection = require('../../config/dbConnection');
 
-exports.addChannel = () => {
+exports.addChannel = (channelName = null) => {
```

Comment:
this does not make any sense here  

```suggestion
exports.addChannel = (channelName ) => {
```

Reply:
we talked about this one, if we send to the query nothing: `null` it wont put a name for the channel

Reply:
i know btw try to test this query alone by sending to it a null value 
i will see it in the morning 
@Amoodaa

Reply:
What is the problem of making the default null? Is it already null?
@Fatmasiam

Reply:
default is undefined. If the channelName is optional, it is good to write a comment on the function explaining that that's by design and not a mistake, also you can name the parameter `channelNameOpt` to make it more clear

Reply:
they say the `undefined` is the default, and you put it as null only when you want it, as a dev, to be `null`